### PR TITLE
Update French translations

### PR DESCRIPTION
**Pull request by frodrigo**

### CODE REVIEW

File: strings.txt [hunk 1]
```diff
@@ -6833,6 +6837,7 @@
     tags = android
     en = Hi! Install MAPS.ME!
     ru = Привет! Установи MAPS.ME!
+    fr = Salut ! Installe MAPS.ME !
```
Comment (Zverik) [inline strings.txt]: There is a space before exclamation marks. Is this normal to French?

File: strings.txt [hunk 2]
```diff
@@ -7001,6 +7006,7 @@
     comment = About short text (below logo)
     tags = ios, android
     en = Joined efforts of Mail.Ru Group and OpenStreetMap make the most useful application for travelers!
+    fr = L'application la plus utile pour les voyageurs, a été créé avec la participation des équipes Mail.Ru et OpenStreetMap.
```
Comment (Zverik) [inline strings.txt]: There will be a merge conflict with https://github.com/mapsme/omim/pull/4159/files#diff-744cbd21eb2f75a5b888f993234e8898R19424

File: strings.txt (hunk 1 above)
Comment (frodrigo) [inline strings.txt]: Yes, before "?", "!" and ":".

File: strings.txt [hunk 3]
```diff
@@ -11623,6 +11629,7 @@
     comment = Showcase menu item.
     tags = ios
     en = Plan your trip
+    fr = planifiez votre voyage
```
Comment (mpimenov) [inline strings.txt]: s/p/P/

File: strings.txt (hunk 2 above)
Comment (mpimenov) [inline strings.txt]: s/Mail.Ru/Mail.Ru Group/ ?

File: strings.txt [hunk 4]
```diff
@@ -6824,6 +6827,7 @@
     comment = Share with friends: menu item title
     tags = android
     en = Tell a Friend
+    fr = Recomander à un ami
```
Comment (mpimenov) [inline strings.txt]: s/m/mm/

File: strings.txt [hunk 5]
```diff
@@ -6420,7 +6420,7 @@
     cs = Plná verze MAPS.ME je dnes zdarma! Stáhněte si ji nyní a dejte vědět přátelům.
     da = Fuld version af MAPS.ME er gratis i dag! Download nu og fortæl dine venner om det.
     nl = Volledige versie van MAPS.ME is nu gratis! Download nu en vertel het je vrienden.
-    fr = Version complète de MAPS.ME est gratuit aujoud'hui ! Téléchargez-le maintenant et dites-le à vos amis.
+    fr = Version complète de MAPS.ME gratuite aujoud'hui ! Téléchargez-le maintenant et dites-le à vos amis.
```
Comment (mpimenov) [inline strings.txt]: s/aujoud'hui/aujourd'hui/

**Comment by frodrigo** [comment]
Thank, merge all your comments !

**Comment by Zverik** [comment]
LGTM

**Comment by mpimenov** [comment]
LGTMME

**Comment by Zverik** [comment]
Thank you!

**Pull request merged**